vector/src/main/scala/geotrellis/vector/GeomFactory.scala

```diff
@@ -53,10 +53,11 @@ private[vector] object GeomFactory extends LazyLogging {
     case s => throw new IllegalArgumentException(s"""Unrecognized JTS precision model, ${precisionType}; expected "floating", "floating_single", or "fixed" """)
   }
 
-  val factory = new geom.GeometryFactory(precisionModel)
+  val coordinateSequenceFactory: CoordinateSequenceFactory = CoordinateArraySequenceFactory.instance
```

Comment:
I though that it's a default coordinate sequence factory; the intention is to override it and expose into public API?